### ISSUE TITLE
npartitions incorrect for n=11160

### ISSUE THREAD

**Issue by brubsby** [sympy/sympy#26750]
implementation is one off for at least 11160, seems like possibly rounding issue over a certain threshold.

```
sympy.__version__
'1.12.1'
sympy.ntheory.npartitions(11160)
62340005729544618156627833313618909586398416985237265626490485213410927284014514633212568209644653901093305326337
```

comparing to pari/gp:
```
? numbpart(11160)
%1 = 62340005729544618156627833313618909586398416985237265626490485213410927284014514633212568209644653901093305326338
```

and a [third source](https://www.asahi-net.or.jp/~KC2H-MSM/mathland/part/part111.txt) confirms pari/gp's result:
```
11160 : 2 * 11 * 21232997492275296875240734393547 * 133454385095703299629360466037236647350566492777318418237199887698594248225147657
```

**Comment by brubsby** [comment]
Looks like it's fixed in 1.13:
```
sympy.__version__
'1.13.0rc3'
sympy.ntheory.npartitions(11160)
<input>:1: SymPyDeprecationWarning: 
The `sympy.ntheory.partitions_.npartitions` has been moved to `sympy.functions.combinatorial.numbers.partition`.
See https://docs.sympy.org/latest/explanation/active-deprecations.html#deprecated-ntheory-symbolic-functions
for details.
This has been deprecated since SymPy version 1.13. It
will be removed in a future version of SymPy.
62340005729544618156627833313618909586398416985237265626490485213410927284014514633212568209644653901093305326338
sympy.functions.combinatorial.numbers.partition(11160)
62340005729544618156627833313618909586398416985237265626490485213410927284014514633212568209644653901093305326338
```

**Comment by oscarbenjamin** [comment]
This was fixed in gh-24293
